packages/core/deploy/008_mock_oracle.js

```diff
@@ -0,0 +1,33 @@
+const { stringToBytes32, interfaceName, ZERO_ADDRESS } = require("@uma/common");
```

Comment:
Is the mock oracle only expected to be deployed on a test L1? Do you think we should have a `prod` tag and a `test` tag or something so we can deploy the entire test setup or the entire prod setup? Also, are you allowed to provide multiple tags on deployment? If so, how do they mix (union or intersection of the tagged scripts?)?

Reply:
Ah yes we should totally use `prod` and `test` tags. I'm thinking we give each contract two tags at least, its name (for use as a dependency in other deployment scripts) and a `prod` tag. This way we can leave out the `prod` tag from the `MockOracle` so we can just run `yarn hardhat deploy --tags prod` to deploy all prod contracts